Modules/zlibmodule.c

```diff
@@ -2015,14 +2015,19 @@ zlib_crc32_combine_impl(PyObject *module, unsigned int crc1,
     return crc32_combine(crc1, crc2, len);
 }
 
+/*[clinic input]
+zlib.__getattr__
+
+    name: object
```

Comment:
```suggestion
    name: str
```
I think we want here a string, isn't?  And remove PyUnicode_Check() in `zlib___getattr__()`

Also, we want this *always*.  Maybe per-default the AC should set an appropriate signature?